src/utilities/index.ts

```diff
@@ -6,7 +6,7 @@ export { convertBuildRuntimesToViewTriggers } from "@utilities/convertBuildRunti
 export { convertTimestampToDate } from "@utilities/convertTimestampToDate.utils";
 export { convertTimestampToEpoch } from "@utilities/convertTimestampToDate.utils";
 export { copyToClipboard } from "@utilities/copyToClipboard.utils";
-export { fetchAndUnpackZip, processReadmeFiles } from "@utilities/fetchAndExtractZip.utils";
+export { fetchAndUnpackZip, processReadmeFiles, processZipContent } from "@utilities/fetchAndExtractZip.utils";
```

Comment:
No need if we're using `fetchAndUnpackZip`

Reply:
removed